atlp-rwanda/champs-ec-fe
# ch(redux):Configuring react redux

**What does this PR do?**

This PR demonstrates the task that was given of setting up  redux in pivot tracker with id #187300180

**Description of Task to be completed?**

-set up Redux store for Next.js app
-modify _app.js to wrap app with Redux Provider
-pass Redux store instance to Provider
-update store configuration based on Redux Toolkit -created welcome slice

How should this be manually tested?
I created a welcome slice , which is consoling the welcome message , you should have to clone the project, then make npm run dev

**What are the relevant pivotal tracker/Trello stories?**

https://www.pivotaltracker.com/n/projects/2698211

**ScreenShot**
![Screenshot 2024-05-27 172636](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/819827d2-dcbf-419a-8937-21d7213dbcfa)
